### ISSUE TITLE
TypeError: Cannot read property 'root' of undefined

### ISSUE THREAD

**Issue by onehorsetown** [tivac/modular-css#248]
Consider this CSS:

```
.foo {
  composes: bar from './bar.css';
}
```

If the file "bar.css" does not exist, the compiler craps out with:

```
node_modules/modular-css/src/processor.js:166
            this._files[dep].result.root.clone(),
                                   
TypeError: Cannot read property 'root' of undefined
    at Promise.all.files.map (.../node_modules/modular-css/src/processor.js:166:36)
    at Array.map (native)
    at Processor.output (.../node_modules/modular-css/src/processor.js:161:34)
    at Compiler.compiler.plugin (.../node_modules/modular-css/src/webpack-plugin.js:19:24)
    at Compiler.applyPluginsAsyncSeries (.../node_modules/tapable/lib/Tapable.js:142:13)
    at Compiler.emitAssets (.../node_modules/webpack/lib/Compiler.js:292:7)
    etc.
```

Using version 3.1.0 with the Webpack 2 plugin.   Nothing fancy.

**Comment by onehorsetown** [comment]
Also, the exact same thing happens with the postcss-import plugin:

```
@import './bar.css';

.foo { }
```

webpack2 plugin configuration:
```
new CSSPlugin({
    css: './styles.css',
    json: './styles.json',
    before: [
      require("postcss-import")
    ]
  })
```

**Comment by tivac** [comment]
Sure, it can handle that a bit more gracefully. Still gonna barf though!

**Comment by tivac** [comment]
Oh, it's because webpack loader errors don't stop compilation. The underlying `modular-css` instance barfs and then webpack marks it as an error and continues on. That's... not what I expected.

**Comment by tivac** [comment]
Fixed in `modular-css@3.2.0`
